app/src/main/java/org/fossasia/susi/ai/activities/LoginActivity.java

```diff
@@ -118,7 +118,7 @@ public void logIn() {
         if(personalServer.isChecked()) {
             if(!CredentialHelper.checkIfEmpty(url,this) && CredentialHelper.isURLValid(url,this)) {
                 if (CredentialHelper.getValidURL(url,this) != null) {
-                    PrefManager.putBoolean("is_susi_server_selected", false);
+                    PrefManager.putBoolean(getApplicationContext().getString(R.string.action_select_server), false);
```

Comment:
@madhavrathi Do that here also. You just did that at one place only. Please change at rest of the places too.